pkgs/development/python-modules/catppuccin-catwalk/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pillow,
+}:
+buildPythonPackage rec {
+  pname = "catppuccin-catwalk";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "catppuccin_catwalk";
+    sha256 = "sha256-5TAw5H3soxe9vLhfj1qs8uMr4ybrHlCj4zdsMzvPo6s=";
+  };
+
+  propagatedBuildInputs = [
+    pillow
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/catppuccin/toolbox";
+    description = "A CLI for Catppuccin that takes in four showcase images and displays them all at once.";
```

```suggestion
    description = "A CLI for Catppuccin that takes in four showcase images and displays them all at once";
```
Please remove a period at the end.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description